Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_Malware_-_Incident_Enrichment.yml

```diff
@@ -0,0 +1,4565 @@
+      version: -1
+      name: Verify Device MAC Address
+      description: |
+        Verify that the ‘devicemacaddress’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "187"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.devicemacaddress
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
+                accessor: MACAddress
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 6970,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "179":
+    id: "179"
+    taskid: 01613bc3-f6fa-4e82-8155-f3aca5aefcf8
+    type: condition
+    task:
+      id: 01613bc3-f6fa-4e82-8155-f3aca5aefcf8
+      version: -1
+      name: Verify Device Name
+      description: |
+        Verify that the ‘devicename’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "186"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: incident.devicename
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
+                accessor: Hostname
+            iscontext: true
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 7370,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "180":
+    id: "180"
+    taskid: 3f7806ed-77f7-4aa6-8c3a-2800336f36b0
+    type: condition
+    task:
+      id: 3f7806ed-77f7-4aa6-8c3a-2800336f36b0
+      version: -1
+      name: Verify Device OS Name
+      description: |
+        Verify that the ‘deviceosname’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "184"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.deviceosname
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
```

Comment:
In case we have 2 endpoints with the same name for example - we might get more than 1 endpoint. and then it will be a list.

Reply:
Replaced all conditions of the 'endpoint context data' check to 'In'